### PR TITLE
General grammar/English fixes

### PR DESCRIPTION
**Pull request by GodyFromDiscord**
This is just a small PR to improve the quality of some sentences and terms used in the package, they provide a clearer understanding and make things look nicer in general. 

This is not a dig at anyone English, but seeing how popular the package is, it seems fitting to make things a nice upstream quality.

### CODE REVIEW

**Review by xHyroM — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -20,15 +20,15 @@ npm i discord-buttons
 
 ## 📜 | Setup
 ```js
-const discord = require('discord.js'); //Define the discord.js module
-const client = new discord.Client(); //Creating discord.js client (constructor)
+const Discord = require('discord.js'); // Define the discord.js module
```
Comment (AngeloCore) [inline README.md:23]: use `discord` not `Discord`

File: README.md [hunk 2]
```diff
@@ -20,15 +20,15 @@ npm i discord-buttons
...
+const client = new Discord.Client(); // Create a discord.js client (constructor)
 const disbut = require('discord-buttons');
 disbut(client);
 ```
 
 ## ✍ | Examples
-All of our Examples and Button Styles are in [examples.md]([http://](https://github.com/AngeloCore/discord-buttons/blob/main/examples.md))
-Also you can check our [Documentation](https://discord-buttons.js.org), if you have any other problems/questions, you can join our [Support Server!](https://discord.gg/5JtyYqW)
+All of our Examples and button styles found in [examples.md]([http://](https://github.com/AngeloCore/discord-buttons/blob/main/examples.md)) and
```
Comment (AngeloCore) [inline README.md:30]: our button styles is a typedef

File: src/index.js [hunk 3]
```diff
@@ -7,17 +7,16 @@ const Message = require('./v12/Classes/Message');
 const { MessageComponentTypes } = require('./v12/Constants');
 
 var version = require('discord.js').version.split('');
-if (version.includes('(')) {
-  version = version.join('').split('(').pop().split('');
-}
+
+if (version.includes('(')) version = version.join('').split('(').pop().split('');
```
Comment (AngeloCore) [inline src/index.js:11]: Do you use prettier?

File: typings/index.d.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import discord, {
+import Discord, {
```
Comment (AngeloCore) [inline typings/index.d.ts:1]: use `discord` everywhere

**Review by AngeloCore — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 3 above)
Comment (GodyFromDiscord) [reply src/index.js:11]: I do not

File: README.md (hunk 1 above)
Comment (GodyFromDiscord) [reply README.md:23]: No, this is the most common usage of importing the discord module.

File: README.md (hunk 2 above)
Comment (GodyFromDiscord) [reply README.md:30]: Nevertheless the grammar is wrong with the capilisation regardless of its type

File: README.md (hunk 1 above)
Comment (JackCrispy) [reply README.md:23]: most common usage is not always the most correct way to format and use code.

Comment (GodyFromDiscord) [reply README.md:23]: In this case, yes it is, Discord itself is a proper noun, meaning it should be capitalised, similar to the discord.js docs when importing the client, the proper noun usage of Discord is used. A lot of examples that work with the module and require it, use capitalisation, it's simple grammar which is what the pull requests is about, so take it with a pinch of salt. It should be capitalised per reasoning of this pull request.

Comment (AngeloCore) [reply README.md:23]: As i said, use `discord`

File: src/index.js (hunk 3 above)
Comment (AngeloCore) [reply src/index.js:11]: Then run `npm run format`

File: README.md (hunk 1 above)
Comment (GodyFromDiscord) [reply README.md:23]: I'll pass on that, and furthermore explained my point, if you simply wont go with it you're just breaching most common uses and cases, so you deny it :)

File: src/index.js (hunk 3 above)
Comment (GodyFromDiscord) [reply src/index.js:11]: Not needed, decreasing the amount of code lines and still works the same way.

File: README.md (hunk 1 above)
Comment (AngeloCore) [reply README.md:23]: Almost all discord.js video tutorials use `discord`. 
So please use `discord`

Comment (GodyFromDiscord) [reply README.md:23]: First 3 videos I found lol https://imgur.com/a/DEWH197 along with the docs themselves, like I said deny the PR if you're so adamant of this.

Comment (AngeloCore) [reply README.md:23]: Use `discord` or we will not merge!...

Comment (GodyFromDiscord) [reply README.md:23]: You've switched it up now man. Just deny the Pull Request like I've said. Not a problem.

Comment (AngeloCore) [reply README.md:23]: Can't you just replace `Discord` with `discord`?

Comment (GodyFromDiscord) [reply README.md]: https://github.com/discord-buttons/discord-buttons/pull/115/commits/8823df6058e627ae32fb47db686b19d604128282

Done.

Comment (GodyFromDiscord) [reply README.md]: also done;

https://github.com/discord-buttons/discord-buttons/pull/115/commits/8a6f24034f6bdb8bef3db5a107eabbcb4c2ce3f5

**Review by AngeloCore — APPROVED** [review]

File: typings/index.d.ts (hunk 4 above)
Comment (AngeloCore) [reply typings/index.d.ts]: ??

**Review by DeltaCoderr — APPROVED** [review]
Seems good 👍🏻

**Review by xHyroM — APPROVED** [review]

**Review by xHyroM — CHANGES_REQUESTED** [review]
@GodyFromDiscord resolve conflicts

**Comment by GodyFromDiscord** [comment]
The conflict has been resolved to fit the new main, but it wont show for some reason, you can see the commit here https://github.com/discord-buttons/discord-buttons/pull/115/commits/8f5ad26608f71fbe6520aae5ea8b52c8c653f10a

**Comment by GodyFromDiscord** [comment]
Just merge it and change it if it keeps being weird

**Review by xHyroM — APPROVED** [review]

**Review by Assassin-1234 — APPROVED** [review]

**Comment by xHyroM** [comment]
Merge?

**Comment by GodyFromDiscord** [comment]
@AngeloCore

**Comment by DeltaCoderr** [comment]
Thanks!

**Pull request merged**